core/roboconf-agent/src/main/java/net/roboconf/agent/internal/misc/AgentConstants.java

```diff
@@ -30,10 +30,10 @@
  */
 public interface AgentConstants {
 
-	String PLATFORM_EC2 = "iaas-ec2";
-	String PLATFORM_OPENSTACK = "iaas-openstack";
-	String PLATFORM_AZURE = "iaas-azure";
-	String PLATFORM_VMWARE = "iaas-vmware";
+	String PLATFORM_EC2 = "@iaas-ec2";
```

Comment:
Changes in config parameters (name and value) probably requires changes in roboconf-system-installers as well (like debian or RH packages) ?